src/viam/sdk/components/camera/camera.hpp

```diff
@@ -25,6 +25,7 @@ namespace sdk {
 /// @ingroup Camera
 class CameraSubtype : public ResourceSubtype {
    public:
+    virtual ~CameraSubtype() = default;
```

Comment:
[nit]: This is equivalent to making the virtual dtor inline. I'd prefer not to do that. I'd move the `=default` into the .cpp file:

In `camera.hpp`:
```
class CameraSubtype ... {
    public:
     virtual ~CameraSubtype();
...
```

In `camera.cpp`:
```
CameraSubtype::~CameraSubtype() = default;
```

No need to change it in this review, but maybe you can enqueue it to your TODOs of small improvements.

Reply:
This is easy enough to do here, gonna take care of it now so we don't have to keep track.